fed/mosaic-output/src/main/resources/web/visualizer-dev.js

```diff
@@ -173,7 +173,7 @@ const Vehicle = {
      */
     createStyle() {
         let style = 'unknown'
-        if (this.vehicleClass === 'Car') {
+        if (this.vehicleClass === 'Car' || this.vehicleClass === 'ElectricVehicle') {
```

Comment:
I did oversee that too in the previous PR. Maybe we should check which of the VehicleClasses could be matched to the car-icon, e.g., `AutomatedVehicle` and `Taxi` should be at least matched, maybe also `HighOccupancyVehicle`.

Reply:
What is HighOccupancyVehicle? Didn't find it here https://sumo.dlr.de/docs/Vehicle_Type_Parameter_Defaults.html

Reply:
At least here it can be found: https://sumo.dlr.de/docs/Definition_of_Vehicles%2C_Vehicle_Types%2C_and_Routes.html#abstract_vehicle_class

I'm not sure if we used that already, but at least in the US HOV-exclusive lanes are quite common.